### ISSUE TITLE
Your ApplyRAUNet freezes gen on DirectML (probably on MPS too) *Solved below*

### ISSUE THREAD

**Issue by MythicalChu** [blepping/comfyui_jankhidiffusion#1]
It's the same problem that happened on IpAdapter and , I think, on base ComfyUI too:

https://github.com/cubiq/ComfyUI_IPAdapter_plus/issues/109#issuecomment-2004516030

I didn't even need to debug your code, just went ahead and altered, on raunet.py, the following lines:

line 26:
sigma = sigma.max().item()
to:
sigma = sigma.max().detach().cpu().numpy()

lines 263 and 277:
sigma = extra_options["sigmas"].max().item()
to:
sigma = extra_options["sigmas"].max().detach().cpu().numpy()

and problem fixed.

**Comment by blepping** [comment]
thanks for the bug report! does the latest commit ( https://github.com/blepping/comfyui_jankhidiffusion/commit/e693373a128156b874677dc3868d7d63e2c7584c ) fix your issue? i took a slightly different approach.

**Comment by MythicalChu** [comment]
It seems to have worked, thanks! (haven't tested extensively yet, but it happened right at 1st gen and now it doesn't anymore, so...)

**Comment by blepping** [comment]
not a problem. let me know if you run into any other issues!
